TSRM/TSRM.c

```diff
@@ -224,21 +224,21 @@ TSRM_API void tsrm_shutdown(void)
 
 	tsrm_reserved_pos  = 0;
 	tsrm_reserved_size = 0;
-}/*}}}*/
+}
 
-/* {{{ */
+/* */
```

Comment:
This doesn't look right; the comment should be removed altogether.

Reply:
Oops, thanks! I'll fix it up manually :) Probably there will be some other small glitches like this.